mozilla/blurts-server
# MNTOR-2347: fxa / subplat instrumentation and utm params

# References: 
Jira: MNTOR-2347




# Description
For
1. user sign-up (via free scan button in landing) and 
2. various subplat signup flows

monitor would want to preserve the UTM parameters in cookies and pass them to FxA. If no UTM is present, then we will pass some default parameters

For user sign up, we'd want to use first-touch strategy for attribution.
For subplat sign up, we'd want to use last-touch strategy for attribution

# Screenshot (if applicable)

Not applicable.

# How to test
* Go to the landing page, and pass in some utm params: ie`http://localhost:6060/?utm_medium=something`
* Try signing up
* Observe the URL for FxA
* Try signing up for subplat in the pricing table
* Observe the URL for FxA

# Checklist (Definition of Done)
